### ISSUE TITLE
Call to api/sign-in?

### ISSUE THREAD

**Issue by sam-wheat** [BitzArt/Blazor.Auth#7]
I followed the [documenation](https://bitzart.github.io/Blazor.Auth/) and created the attached sample app.  
It appears the app is making a call to /api/sign-in, which is failing.  Is there a a way to prevent this?

![image](https://github.com/BitzArt/Blazor.Auth/assets/16885210/8c581ee3-842d-4f81-a5ac-228929ff5eab)

[Bitzart.Blazor.Auth.Example.zip](https://github.com/BitzArt/Blazor.Auth/files/15283382/Bitzart.Blazor.Auth.Example.zip)

**Comment by YuriyDurov** [comment]
@sam-wheat Thank you for reaching out!

I have checked the solution you attached. Indeed, the implementation guide was missing an important part about mapping auth endpoints that would allow your client project to access your server-side authentication logic. It is now [added](https://bitzart.github.io/Blazor.Auth/03.authentication.html#register-the-authentication-service) to the documentation.

Here is what was added:

> Add the authentication endpoints in your Blazor Server Program.cs. This is required in order to allow the Client project to call the Server's API and use the server-side IAuthenticationService implementation.

```csharp
// Program.cs
app.MapAuthEndpoints();
```

After adding this line to your server-side Blazor `Program.cs`, the solution seems to be working. Do let me know if this resolves the issue 😊
